docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,484 @@
+      "progress" : [
+        {
+          "phase" : "reindexing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "loading_data",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "analyzing",
+          "progress_percent" : 100
+        },
+        {
+          "phase" : "writing_results",
+          "progress_percent" : 100
+        }
+      ]
+    }
+  ]
+}
+----  
+
+The job has four phases. When all the phases have completed, the job 
+state becomes `stopped` and the results are ready to view and evaluate.
+--
+
+
+[[flightdata-classification-results]]
+==== Viewing results
+
+. Use the standard {es} search command to view the results in the destination 
+index:
++
+--
+
+[source,console]
+--------------------------------------------------
+GET df-flight-results/_search
+--------------------------------------------------
+// TEST[skip:TBD]
+
+
+The snippet below shows a part of a document with the annotated results:
+
+[source,console-result]
+----  
+          ...
+          "FlightDelay" : false, <1>
+          "DistanceMiles" : 8865.200239823978,
+          "FlightTimeMin" : 792.6198230421824,
+          "OriginWeather" : "Cloudy",
+          "dayOfWeek" : 1,
+          "AvgTicketPrice" : 816.1058877497752,
+          "Carrier" : "Kibana Airlines",
+          "FlightDelayMin" : 0,
+          "OriginRegion" : "CO-CUN",
+          "FlightDelayType" : "No Delay",
+          "DestAirportID" : "NRT",
+          "timestamp" : "2019-11-12T17:06:35",
+          "Dest" : "Narita International Airport",
+          "FlightTimeHour" : 13.210330384036373,
+          "Cancelled" : false,
+          "DistanceKilometers" : 14267.156814759282,
+          "OriginCityName" : "Bogota",
+          "DestWeather" : "Sunny",
+          "OriginCountry" : "CO",
+          "ml__id_copy" : "-7OQg24B95sXiIrm0EZo",
+          "DestCountry" : "JP",
+          "DestRegion" : "SE-BD",
+          "OriginAirportID" : "BOG",
+          "DestCityName" : "Tokyo",
+          "ml" : {
+            "top_classes" : [ <2>
+              {
+                "class_probability" : 0.939335365058496, <3>
+                "class_name" : "false" <4>
+              },
+              {
+                "class_probability" : 0.06066463494150393,
+                "class_name" : "true"
+              }
+            ],
+            "FlightDelay_prediction" : "false", <5>
+            "is_training" : false <6>
+          }
+----
+<1> The `dependent_variable` with the ground truth value. This is what we are 
+trying to predict with the {classanalysis}.
+<2> The top classes object contains the predicted classes with the highest 
+probability.
+<3> The probability that the datapoint belongs to the given class. This is a 
+value between 0 and 1. The higher the number the higher the probability that the 
+datapoint belongs to the class.
+<4> Name of the class.
+<5> The prediction. The field name is suffixed with `_prediction`.
```

Comment:
This is true unless the user specified `prediction_field_name` via the API. In such a case, there will be no `_prediction` suffix.